src/partials/toc.html

```diff
@@ -23,7 +23,7 @@
 {% import "partials/language.html" as lang with context %}
 
 <!-- Table of contents -->
-<nav class="md-nav md-nav--secondary">
+<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
```

Comment:
Please use the translation `"toc.title"` here. See the base template for how to use translations.